README.md

```diff
@@ -81,7 +81,7 @@ The following settings must be passed as environment variables as shown in the e
 
 ## Action outputs
 
-| name               | description                                                                             |
-| ------------------ | --------------------------------------------------------------------------------------- |
-| `object_key`       | Uploaded object key generated by [shortid](https://github.com/dylang/shortid) in Bucket |
-| `object_locations` | Object Locations                                                                        |
+| name               | description                                                                                   |
+| ------------------ | --------------------------------------------------------------------------------------------- |
+| `object_key`       | Uploaded object key generated by [shortid](https://github.com/dylang/shortid) in Bucket       |
+| `object_locations` | The URLs of the uploaded objects as a JSON string. E.g: `["location/file1","location/file2"]` |
```

Comment:
why use the array to string?? 

this is intended to control by pragmatically in other actions

Reply:
@kittaakos you're right. 
must have to stringify Object. 

but core was update this https://github.com/actions/toolkit/issues/370

so could you update @actions/core?